cli/organizationroles.go

```diff
@@ -203,7 +217,12 @@ func (r *RootCmd) editOrganizationRole(orgContext *OrganizationContext) *serpent
 				// Do not actually post
 				updated = customRole
 			} else {
-				updated, err = client.PatchOrganizationRole(ctx, customRole)
+				switch createNewRole {
+				case true:
+					updated, err = client.CreateOrganizationRole(ctx, customRole)
+				default:
+					updated, err = client.UpdateOrganizationRole(ctx, customRole)
+				}
```

Comment:
This feels like a smell to me, my gut would say that we should just have a separate `create` cli command. If the UI needs to make the distinction, sounds like we should do the same for the CLI.

Reply:
You are right @f0ssel. This PR is already quite big, and we are not launching this yet. Can I create an issue to split these out into 2 commands and do it in another PR?

Reply:
Totally 👍